### DISCUSSION TITLE
POD ERROR

### DISCUSSION THREAD

**Discussion by Ehsan-Nawaz-Dev** [react-native-community/cli#2210] [Q&A]
Can Anyone help me this error i search a lot but got nothing about this error my react-native version is 0.70.0 its show this error when i install a fresh and latest version metro doesn't start automatically my OS is windows 10

![error](https://github.com/react-native-community/cli/assets/89260855/39f0d928-1f04-4fe5-b8f8-914af3f6bdbf)

**Comment by JimmyDaddy** [comment]
Maybe you should set the `automaticPodsInstallation` to `false` in your project's `react-native.config.js` file located at `{yourProjectRoot}/react-native.config.js`.

**Comment by szymonrybczak** [comment]
Duplicate of https://github.com/react-native-community/cli/issues/2211.
